drivers/can/can_mcp2515.c

```diff
@@ -296,16 +387,16 @@ static int mcp2515_send(struct device *dev, const struct zcan_frame *msg,
 	dev_data->tx_cb[tx_idx].cb = callback;
 	dev_data->tx_cb[tx_idx].cb_arg = callback_arg;
 
-	addr_tx_ctrl = MCP2515_ADDR_TXB0CTRL +
-		       (tx_idx * MCP2515_ADDR_OFFSET_FRAME2FRAME);
-
 	mcp2515_convert_zcanframe_to_mcp2515frame(msg, tx_frame);
-	mcp2515_cmd_write_reg(dev,
-			      addr_tx_ctrl + MCP2515_ADDR_OFFSET_CTRL2FRAME,
-			      tx_frame, sizeof(tx_frame));
+
+	/* Address Pointer selection */
+	u8_t abc = 2 * tx_idx;
```

Comment:
You need to define variables on top of the function

Reply:
It's the same for nnn, just in case

Reply:
Done